sklearn/linear_model/logistic.py

```diff
@@ -999,7 +999,7 @@ class LogisticRegression(BaseEstimator, LinearClassifierMixin,
         To lessen the effect of regularization on synthetic feature weight
         (and therefore on the intercept) intercept_scaling has to be increased.
 
-    class_weight : dict or 'balanced', optional
+    class_weight : dict or 'balanced', default: None
```

Comment:
The discussion got hidden, but I think usually we have either a default value or "optional".
All parameters are optional. There is a common test for that ;)
Optional rather means "The default parameter is None or zero". I wouldn't specify both the default, and optional, and I prefer specifying the default, but there is no agreement on that and it is by no means consistent across the code base.